modules/mono/editor/GodotTools/GodotTools/Ides/MessagingServer.cs

```diff
@@ -364,6 +368,7 @@ private static Task<Response> HandleDebugPlay(DebugPlayRequest request)
 
             private static Task<Response> HandleStopPlay(StopPlayRequest request)
             {
+                _ = request;
                 DispatchToMainThread(Internal.EditorRunStop);
                 return Task.FromResult<Response>(new StopPlayResponse());
             }
```

Comment:
Is this just to avoid a warning because `request` is otherwise not used? I wonder if there is a better way to handle this.

Reply:
Not a more readable way that I know of. In this particular case we could remove the parameter, but there are other cases with virtual methods and such.